README.md

```diff
@@ -74,6 +74,8 @@ The required version of Xcode changes frequently, and is often a beta release.
 Check this document or the host information on <https://ci.swift.org> for the
 current required version.
 
+In Xcode Preferences, on the Locations tab, Command Line Tools should be set to Xcode 10.0 (10L176w).
```

Comment:
I think we ought to either mention `xcode-select` or provide a sample command.  The individual build numbers of Xcodes are too volatile to put in this document.

Reply:
Thank you.  Your proposed approach is better.

Reply:
I've committed a revised version.  Thanks, again, Robert.